systemd-units/meson.build

```diff
@@ -16,3 +17,9 @@ install_data(
   'hirte-agent.service',
   install_dir : systemd_unit_dir
 )
+
+install_data(
+  ['hirte-agent-user.service'],
+  rename  : ['hirte-agent.service'],
```

Comment:
Just saw that the RPM build fails. This file needs to be added to the hirte-agent file list in `hirte.spec.in`